Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.8
+
+RUN apk add --no-cache python3
+RUN pip3 install --upgrade pip
+
+#RUN pip3 install --no-cache terminal-translator
+
+#for Dev
+ADD . /translate-in-terminal
+WORKDIR /translate-in-terminal
+RUN pip3 install wheel
+RUN python3 setup.py bdist_wheel
+RUN pip3 install dist/terminal_translator-1.0.1-py3-none-any.whl -I
```

Comment:
Can not you change the _version (1.0.1)_ part to a variable? `translate/__init__.py`.

Reply:
Change build steps.

d532d995b87f8833cbe43841488a5515194c74ac